components/calendar/src/types.rs

```diff
@@ -293,21 +293,32 @@ dt_unit!(
     IsoHour,
     u8,
     24,
-    "An ISO-8601 hour component, for use with ISO calendars.\n\nMust be within inclusive bounds `[0, 24]`."
+    "An ISO-8601 hour component, for use with ISO calendars.
+
+Must be within inclusive bounds `[0, 24]`. The value could be equal to 24 to
+indicate the end of a day, with the writing 24:00:00."
 );
 
 dt_unit!(
     IsoMinute,
     u8,
     60,
-    "An ISO-8601 minute component, for use with ISO calendars.\n\nMust be within inclusive bounds `[0, 60]`."
+    "An ISO-8601 minute component, for use with ISO calendars.
+
+Must be within inclusive bounds `[0, 60]`. The value could be equal to 60 to
+indicate the end of an hour, with the writing 12:60:00."
```

Comment:
Like the note above for hours, this clarification of the semantics is a good idea.

Contrary to the hours above (for which the meaning of 24 is defined by Section 4.2.3 in ISO 8601:2004), this is an extension to ISO 8601; this should be noted, and a similar comment should be made for seconds (with 70 being changed back to 61, see my other comment).